language/stdlib/modules/LibraTransaction.move

```diff
@@ -0,0 +1,213 @@
+address 0x1 {
+
+module LibraTransaction {
```

Comment:
This is the newly created module for transaction prologues and epilogues (what used to be `LibraWriteSetManager`).